src/deployers/PluginsDeployerFactory.h

```diff
@@ -23,6 +23,7 @@ namespace linuxdeploy {
                 const boost::filesystem::path qtInstallQmlPath;
                 const boost::filesystem::path qtTranslationsPath;
                 const boost::filesystem::path qtDataPath;
+                const int qtMajorVersion{5};
```

Comment:
I think we should not use a default value. You keep track of the number anyway where the deployers are called.

Reply:
Applied